.eslintrc

```diff
@@ -85,6 +85,9 @@
     "prettier/prettier": "error",
     "react/jsx-filename-extension": "off",
     "import/no-extraneous-dependencies": "off",
+    "no-restricted-imports": "off",
+    "no-unused-vars": "off",
+    "@typescript-eslint/consistent-type-imports": "off",
```

Comment:
Can we add a comment here to explain the reason for disabling with a todo or a ticket number to reference for future enablement of these?